src/file-uploader/__tests__/utils/get-file-data-transfer-items.test.js

```diff
@@ -0,0 +1,38 @@
+const { MimeType } = require('../../../constants')
+const { default: getFileDataTransferItems } = require('../../src/utils/get-file-data-transfer-items')
```

Comment:
VS Code kept auto-importing stuff with `require` instead of `import` - must have missed this one. I'll clean it up quick!